mealie/routes/admin/admin_invitations.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+
+from mealie.routes._base import BaseAdminController, controller
+from mealie.schema.household.invite_token import (
+    ReadInviteToken,
+)
+from mealie.schema.make_dependable import make_dependable
+from mealie.schema.response.pagination import PaginationQuery
+
+router = APIRouter(prefix="/invitations")
+
+
+@controller(router)
+class AdminInvitationsController(BaseAdminController):
```

Comment:
Maybe I'm just missing it, but where in the frontend is this endpoint being called? I only see us using the old (non-admin) route

Reply:
This admin route is not called via the frontend just like the non-admin one to get all the created invitations.
There is no page associated with these operations and they are only available via API. The admin route was added to give access to all groups/households, not the current user's.
I don't think an admin page containing all current or past invitation links would be useful on the frontend side.
Also, I don't think it is necessary to duplicate the endpoint to create an invitation on the admin controller?

Reply:
Yeah I agree that would be unnecessary. Let's go ahead and remove the new backend route/controller then if it's not being used in the app